doc/man3/OPENSSL_malloc.pod

```diff
@@ -203,6 +210,16 @@ OPENSSL_mem_debug_push(), OPENSSL_mem_debug_pop(),
 CRYPTO_mem_debug_push(), and CRYPTO_mem_debug_pop()
 are deprecated and are no-ops that always return 0.
 
+OPENSSL_strtoul() returns 1 on success and 0 in the event that an error has
+occured.  Note that a success condition does not imply that the expected
+translation has been preformed.  For instance calling
+
+    OPENSSL_strtoul("0x12345", &endptr, 0, &num);
+
+will result in a successful translation with num having the value 0, and
```

Comment:
`I<num>`

Reply:
And actually, this is wrong - with a `base` of `0` the leading `0x` will be interpreted as "the following is in hexadecimal", and you do actually get 0x12345 (I've tested locally to be sure!). This should go in as a test case

Reply:
(to get the example you want, change the base from `0` to `10`)

Reply:
@t8m , yep, docs were wrong. requested test cases are already available:
```
/* pass on conv "0x12345" to 0x12345, base 16 */
    {
        "0x12345", 0, 0x12345, 1, 7
    },
/* pass on base 10 translation, endptr points to 'x' */
    {
        "0x12345", 10, 0, 1, 1
    },
```